ChemoTracker/components/PatientEducation/Search.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from 'react';
+import { ActivityIndicator, FlatList, Linking, ListView, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import { List, ListItem } from "react-native-elements";
+import { TabNavigator } from 'react-navigation';
+import { WebBrowser } from 'expo';
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome';
+import IonIcon from 'react-native-vector-icons/Ionicons';
+import { searchStyles } from '../../styles/PatientEducation/patient-education.js';
+import color from '../../styles/color.js';
+
+const api = 'http://ec2-52-15-106-40.us-east-2.compute.amazonaws.com:8000';
+const header = {
+  'Accept': 'application/json',
+  'Content-Type': 'application/json'
+}
+
+export const getSearchResults = (searchQuery) => {
+  const queryparams = `query=${searchQuery}`;
+  return fetch(`${api}/patient-education/posts?${queryparams}`, {
+    method: 'get',
+    headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+    }
+  }).then(res => {
+    return res.json();
+  });
+}
+
+class Search extends Component {
+	static navigationOptions = {
+		tabBarLabel: "Search",
+		tabBarIcon: () => (<FontAwesomeIcon size={24} name="search" color={color.navBarIcon} />)
+  }
+
+  searchQuery = this.props.navigation.state.params.searchQuery;
+  
+  state = {
+    loading: false,
+    data: [],
+    page: 1,
+    error: null,
+    refreshing: false
+  };
+  
+  componentDidMount() {
+    this.makeRemoteRequest();
+  }
+
+  makeRemoteRequest = () => {
+    const { page } = this.state;
+    getSearchResults(this.searchQuery)
+      .then(res => { 
+        this.setState({
+          data: page === 1 ? res.posts : [...this.state.data, ...res.posts],
+          error: res.error || null,
+          loading: false,
+          refreshing: false
+        });
+      })
+      .catch(error => {
+        this.setState({ error, loading: false });
+      });
+  }
+
+  handleRefresh = () => {
+    this.setState(
+      {
+        page: 1,
+        refreshing: true
+      },() => {
+        this.makeRemoteRequest();
+      }
+    );
+  };
+
+  handleLoadMore = () => {
+    // this.setState(
```

Comment:
delete if not going to be used

Reply:
leaving this too for future to be able to load more results